java/com/google/location/lbs/activity/audioset/dataprocessing/incomplete-example.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
```

Comment:
we typically use "_" not "-" in file names

Reply:
Ah. Thank you!